crates/context/config/src/client/protocol/starknet.rs

```diff
@@ -248,7 +250,9 @@ impl Network {
             |result| {
                 Ok(result
                     .into_iter()
-                    .flat_map(|felt| felt.to_bytes_be().to_vec())
+                    // Remove the skip(1) here
+                    .map(|felt| felt.to_bytes_be())
+                    .flatten()
```

Comment:
is it still coming or it's already gone?

Reply:
updated